Rakefile

```diff
@@ -16,7 +16,7 @@ end
 task compile: :make
 task compile_no_debug: :make_no_debug
 
-task default: [:clobber, :compile, :test]
```

Comment:
If one wants to always clean before, it's possible via `bundle exec rake clobber default`